plugins/outputs/influxdb_v2/errors.go

```diff
@@ -0,0 +1,51 @@
+package influxdb_v2
+
+import (
+	"fmt"
+	"time"
+)
+
+type genericRespError struct {
+	Code      string
+	Message   string
+	Line      *int32
+	MaxLength *int32
```

Comment:
Why int32?
```suggestion
	Line      *int
	MaxLength *int
```

Reply:
Let's not change this as this code was there before the PR.